_news-and-blog/2023-10-17-solutions-by-families-for-families.md

```diff
@@ -0,0 +1,250 @@
+
+**2. Families face administrative barriers applying to programs.**
+
+_“When you call to try to talk to somebody, you never can get through. […] If I leave her a message, it’s probably that I have to leave her six messages before she’ll ever call me back. And it’s two or three weeks before they will call me back.”_
+
+The application processes for expecting and new parents can be time-consuming and resource-intensive, with siloed
+procedures adding to the burden. Additionally, limited virtual options and inconvenient office hours prevent parents
+from being able to apply at a time that works for their work or transportation schedules.
+
+**3. Families lack key physical goods — like diapers and breast pumps — after giving birth.**
+
+_“The baby came 2 weeks early. We hadn’t gotten a car seat yet because we were saving up for it. But babies don’t care if you don’t get paid till Friday.”_
+
+The cost of critical family supplies, such as diapers and breast pumps, fall outside most benefits program coverage, and reimbursement can be slow and cumbersome for items that Medicaid covers.
+
+**4. Families face barriers to maintain benefits as they juggle the responsibilities of parenthood.**
+
+_“My last SNAP re-enrollment letter got lost in the mail, and we ended up going 3 weeks without food stamps while the paperwork got in order.”_
+
+Benefits program communications about re-enrollment requirements and timelines can be confusing, especially as they
+often rely heavily on mail-based communications methods.
+
+## What’s next?
+
+As we implement the design phase, we’ll continue to use a human-centered design approach that drives how we design,
+launch, and deliver public services.
+
+Based on the insights we’ve gained from innovative community leaders and families, we are confident our team, working
+with communities and our pilot partners, can improve the way the federal government serves families in the early years.
+
+Learn more about our pilot solutions and projects, and how we’re testing them through continued co-design with families
+at the [Having a child and early childhood](https://www.performance.gov/cx/life-experiences/having-a-child-and-early-childhood-for-low-income-families/) life customer experience page.
+
+~
+
+_This blog is part two in a four-part series that highlights the “Birth and early childhood” Life Experience team’s
+work. Brought together by the Customer Experience Executive Order, teams are tackling the extraordinary challenge of
+modernizing and implementing government services that are simple, accessible, easy to use, and equitable for families._
+
+_Read the team’s first post, “Why We’re Designing Government to Work Better and Smarter for Families” on [our blog](/blog/why-were-designing-government-to-work-better-and-smarter-for-families)._
```

Comment:
Same comment from line 24:
update link for our blog to: https://usds.github.io/website-staging/news-and-blog/why-were-designing-government-to-work-better-and-smarter-for-families-dee5c2